staging/src/k8s.io/client-go/tools/cache/controller.go

```diff
@@ -167,9 +167,9 @@ func (c *controller) processLoop() {
 //  * OnUpdate is called when an object is modified. Note that oldObj is the
 //      last known state of the object-- it is possible that several changes
 //      were combined together, so you can't use this to see every single
-//      change. OnUpdate is also called when a re-list happens, and it will
-//      get called even if nothing changed. This is useful for periodically
-//      evaluating or syncing something.
+//      change. OnUpdate is also called when a re-list and resync happens, and
+//      it will get called even if nothing changed. This is useful for
```

Comment:
this sentence is trying to say, that in the case of resync, it will get called even though nothing has changed, but needs rewording

Reply:
hm ,it says exactly that: "it will get called even if nothing changed", no? :)

Reply:
`OnUpdate is also called when a re-list and resync happens, and  it will get called even if nothing changed.`, in the existing setence the `it` you are assuming qualifies the resync , but that is not clear. i would say `OnUpdate is also called when a re-list and resync happens. For a resync, OnUpdate  is called even if nothing changed.`